Dockerfile

```diff
@@ -21,4 +24,6 @@ RUN npm run build
 # Copy the rest of the app
 COPY server /app/server/
 
-ENTRYPOINT ["npm" "start"]
+HEALTHCHECK CMD curl --fail http://localhost:3050/health-check
```

Comment:
Lots of the examples out there add a `|| exit 1` to the end. Does the Docker `HEALTHCHECK` really not play well with non-`1` error codes?

Reply:
Maybe:

> - 0: success - the container is healthy and ready for use
> - 1: unhealthy - the container is not working correctly
> - 2: reserved - do not use this exit code
>
> *-- https://docs.docker.com/engine/reference/builder/#healthcheck*

We could move to a custom health check Node.js script and ditch the `curl`

Reply:
Moved to a Node.js script and removed the `curl` dependency ⏩